cukedoctor-converter/src/main/java/com/github/cukedoctor/util/builder/FeatureBuilder.java

```diff
@@ -81,146 +87,134 @@ public Feature build() {
 
   public Feature aFeatureWithTwoScenarios() {
```

Comment:
I can't remember why these `aFeatureWith*` are in main source, I think we should move to test folder

Reply:
Well spotted! Moved that to test scope in a utility class called Features